modules/extensions-spring-support/src/test/resources/models/values.json

```diff
@@ -11034,7 +11034,7 @@
           }
         }
       },
-      "minMuleVersion": "4.1.1",
+      "minMuleVersion": "4.5.0",
```

Comment:
Method multiLevelValue has min mule version 4.5 because of its parameter optionsParameter. Parameter optionsParameter has min mule version 4.5 because it is of type GroupAsMultiLevelValue. Type GroupAsMultiLevelValue has min mule version 4.5 because of its field country. Field country has min mule version 4.5 because it is annotated with ValuePart. ValuePart was introduced in Mule 4.5.